python/spark_sklearn/converter_test.py

```diff
@@ -72,7 +72,6 @@ def ztest_toPandas(self):
 @fixtureReuseSparkSession
 class CSRVectorUDTTests(MLlibTestCase):
 
-    @unittest.skip("CSR Matrix support not present for Spark 2.0 - see issue #24")
```

Comment:
Note that this demonstrates that https://github.com/databricks/spark-sklearn/issues/24 was resolved earlier